Fairly balances the Bingle threat

Zoooooooop opened the pull request:
About The Pull Request

Increases the stamina attack to negative 50, which is 4 hits to stun (I think)
Slightly increases the weight to 8, which is still lower than the pre nerf

 Why It's Good For The Game

Look, a lot of people actually love this antag and the nerfs just don't make sense to me. It feels like someone doesn't like this antag or got salty and over did the nerfs.  I don't think the game needs to revolve around heavy handed nerfs and sometimes the solution is literally get better.

The original nerf made it to be 6 hits to stamina crit which is way to much.  It's hard to obtain that in combat and you'll just likely end up dying. 4 is a decent number since you still need to put effort in to down crew and the crew can still run away... unless they made a bad play and get punished. In fact, I think I've seen people get killed more than actually get stamina crit from these things.

I also increased the rate a bit to make them pop up more, but not as much as before.

After the original nerf [https://github.com/Monkestation/Monkestation2.0/pull/7599](url) bingles have been struggling to get anywhere. They are often very short lived and easy to deal with. Give them some bite back or remove them from the game entirely.

TheColorCyan commented:
>It feels like someone doesn't like this antag or got salty and over did the nerfs. 

person that did the nerfs is the same person that ported bingles in the first place

>Give them some bite back or remove them from the game entirely.

I wouldn't mind if they got removed
We already have a lot of high threat no/low rp antags added recently (darkspawn and bloodlings for example)
bingles fall into same category where they bring nothing to the round but destruction, and getting thrown into the pit is either live in hell, afk, or just ghost out of your body
fighting them is also in general unfun even with nerfs

notghosti reviewed (commented):
> Increases the stamina attack to negative 50, which is 4 hits to stun (I think)
We run off of 250 stamina which is 5 hit-ish
As for the changes themselves. I can live with our without the stamina changes. But the weight of a bingle lord does not need to be increased. 5 is also the weight of a spider infestation which is a similar esk threat, and they at least still have to obliged but rules of medics, pressure, ect.

Zoooooooop commented:
I'll remove the weight changes and adjust the stamina hold on

Zoooooooop commented:
> Want me to add a changelog? As thats the only functional issue with the pr atm.

I just added one. I also fixed up the "why" section as I think I was just overly mean first time around.

Veth-s commented:
> > Want me to add a changelog? As thats the only functional issue with the pr atm.
> 
> I just added one. I also fixed up the "why" section as I think I was just overly mean first time around.

Can you update the changelog to read:
"Increased bingle attack stamina damage to -65 (up from -35)"
please :)

ChiefMonkengineer commented:
<!-- tgs_test_merge_comment -->
## Test merge deployment history:

<details><summary>Test Merge Deployed @ 08/31/2025 23:14:07 +00:00:</summary>

##### Server Instance
Monkestation

##### Merged By
notghosti

##### Revision
Origin: b287e6e70641bf6e7584cfdbb890cbed330e711d
Pull Request: 20ffae3b616f644c3c84d2e18baf6421ac58a43c
Server: 5c98bd5b70011e806c086da99b5fc25ca1519596
</details>

Zoooooooop commented:
> > > Want me to add a changelog? As thats the only functional issue with the pr atm.
> > 
> > 
> > I just added one. I also fixed up the "why" section as I think I was just overly mean first time around.
> 
> Can you update the changelog to read: "Increased bingle attack stamina damage to -65 (up from -35)" please :)

Alright just did so.
